docs/intro.rst

```diff
@@ -33,8 +33,6 @@ If you are using Windows, then the following should be used instead: ::
 
 To install additional packages for speedup,  you should use ``py-cord[speed]`` instead of ``py-cord``, e.g. ::
 
-.. code:: sh
```

Comment:
how does this fix it?

Reply:
^this btw

Reply:
because, see this (the original)
![image](https://user-images.githubusercontent.com/68941022/145700932-8232cfdc-36fc-4999-807f-b3da02d51117.png)


and this is the one i fixed
![image](https://user-images.githubusercontent.com/68941022/145700937-2c29bea2-9809-461f-849d-80e09a9d3f81.png)

Reply:
this makes it a non-codeblock though?, plus it doesn't actually show for me

Reply:
> this makes it a non-codeblock though?, plus it doesn't actually show for me

actually i don't know but the code block thing ain't working, also it is [showing for me](https://github.com/agam778/pycord/blob/f39401341eb32ffee98b6ca65c6d4560914e004d/docs/intro.rst) without any issues tho lol